frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadFileForm.component.tsx

```diff
@@ -58,13 +60,17 @@ export const UploadFileForm = ({
 }: IUploadFileForm): JSX.Element => {
 	const teamspace = TeamspacesHooksSelectors.selectCurrentTeamspace();
 	const project = ProjectsHooksSelectors.selectCurrentProject();
+	FederationsActionsDispatchers.fetchFederations(teamspace, project);
+
 	const [selectedIndex, setSelectedIndex] = useState<number>(null);
 	const [isUploading, setIsUploading] = useState<boolean>(false);
 	const methods = useForm<UploadFieldArray>({
```

Comment:
For some reason the object being passed into onSubmit() is now empty. I've had a quick look and can't figure out why. But it means none of the uploads work